dummyfs/dummyfs.c

```diff
@@ -160,9 +160,19 @@ int dummyfs_setattr(void *ctx, oid_t *oid, int type, long long attr, const void
 			/* TODO: add mouting capabilities */
 			ret = -EINVAL;
 			break;
+
+		case (atMTime):
+			o->mtime = attr;
+			break;
+
+		case (atATime):
+			o->atime = attr;
+			break;
 	}
 
-	o->mtime = time(NULL);
+	/* FIXME: mtime and atime are always set together */
+	if (type != atMTime && type != atATime)
+		o->mtime = time(NULL);
```

Comment:
Hmm, maybe we want to do this when atATime? Or at least do o->mtime = o->atime?

Reply:
utimes is always setting both atime and mtime so we don't want to overwrite it. We should probably pass both values atomically, that's why I've added FIXME.